QUARKUS-3249.md

```diff
@@ -0,0 +1,32 @@
+# QUARKUS-3249 Configurable init container
+
+This document is a high level test plan for options to configure init containers in Quarkus 
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-3249
+
+## Scope of the testing
+* Check, that flyway task can be disabled
```

Comment:
Is this strictly related to flyway? Flyway itself is not supported in Quarkus, but I see there's some Kafka in the issue description. Do you know what the status on that is?

Is there any way to affect kube init pods while using supported extensions?